### PR TITLE
OEL-1290: Field type replaced by oe_featured_media on events.

### PR DESCRIPTION
**Pull request by Maxfire**
Field type modified on events.

### CODE REVIEW

File: modules/oe_starter_content_event/config/install/core.entity_view_display.node.oe_sc_event.default.yml [hunk 1]
```diff
@@ -41,13 +41,12 @@ content:
...
   oe_featured_media:
-    type: entity_reference_entity_view
+    type: oe_featured_media_label
     label: above
     settings:
-      view_mode: default
       link: true
     third_party_settings: {  }
-    weight: 3
+    weight: 102
```
Comment (brummbar) [inline modules/oe_starter_content_event/config/install/core.entity_view_display.node.oe_sc_event.default.yml]: The weight seems off, the field was supposed to show higher in the page. Better re-save the form display and re-export.

File: tests/src/Functional/NewsIntegrationTest.php [hunk 2]
```diff
@@ -79,7 +79,7 @@ public function testCreateNews() {
     $page->fillField('Introduction', 'Example Introduction');
     $page->fillField('Date', '2022-01-24');
     $media_name = $media_image->getName() . ' (' . $media_image->id() . ')';
-    $page->fillField('Use existing media', $media_name);
+    $page->fillField('Media item', $media_name);
```
Comment (brummbar) [inline tests/src/Functional/NewsIntegrationTest.php:82]: Here we need to fill also the caption field, which this new "featured media" field type has.

File: tests/src/Functional/NewsIntegrationTest.php [hunk 3]
```diff
@@ -88,9 +88,7 @@ public function testCreateNews() {
     $assert_session->pageTextContains('Example Content');
     $assert_session->pageTextContains('Example Introduction');
     $assert_session->pageTextContains('01/24/2022');
-    $assert_session->responseContains('image-test.png');
```
Comment (brummbar) [inline tests/src/Functional/NewsIntegrationTest.php:91]: We can't assert the image itself but we can assert that there's the title of the media and its caption rendered.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: tests/src/Functional/NewsIntegrationTest.php (hunk 3 above)
Comment (Maxfire) [reply tests/src/Functional/NewsIntegrationTest.php:91]: Nice call, I did for Events as well.

File: tests/src/Functional/NewsIntegrationTest.php (hunk 2 above)
Comment (Maxfire) [reply tests/src/Functional/NewsIntegrationTest.php:82]: Nice call, I did for Events as well.

**Review by escuriola — DISMISSED** [review]

**Review by brummbar — APPROVED** [review]

**Review by escuriola — APPROVED** [review]

**Pull request merged**